### PR TITLE
Added tellor360 contractset

### PR DESCRIPTION
**Pull request by akremstudy**
For oracle and autopay, added:
- contract addresses to contract_directory
- abi
- and created the 360 contractset (oracle, autopay, and token(existing).

### CODE REVIEW

**Comment by akremstudy** [comment]
@oraclown can you please take a look at the tox style run, I haven't seen that error before. Did something change in the way the test is ran?

File: src/telliot_core/apps/core.py [hunk 1]
```diff
@@ -123,6 +132,27 @@ def get_tellorx_contracts(self) -> TellorxContractSet:
 
     _tellorx: Optional[TellorxContractSet]
 
+    def get_tellor360_contracts(self) -> Tellor360ContractSet:
+        """Get or create TellorX contracts"""
```
Comment (oraclown) [inline src/telliot_core/apps/core.py:136]: It says tellorx

File: src/telliot_core/tellor/tellor360/autopay.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+logger = logging.getLogger(__name__)
+
+
+class Tellor360AutopayContract(Contract):
+    def __init__(self, node: RPCEndpoint, account: Optional[ChainedAccount] = None):
+        chain_id = node.chain_id
+        assert chain_id is not None
+
+        contract_info = contract_directory.find(chain_id=chain_id, name="tellor360-autopay")[0]
```
Comment (oraclown) [inline src/telliot_core/tellor/tellor360/autopay.py:22]: Will throw an index error if there's no 260 contracts for the desired chain

**Review by oraclown — CHANGES_REQUESTED** [review]
Thanks for this @akremstudy ! Think any unit tests should be added to make sure everything's working fine now and when changes are made in the future?

**Comment by akremstudy** [comment]
> Thanks for this @akremstudy ! Think any unit tests should be added to make sure everything's working fine now and when changes are made in the future?

I don't think deploying to brownie and testing is necessary since no functions are added to the files and contract read or write are tested already but if you think it is let me know.

**Pull request merged**